fuzz/README.md

```diff
@@ -10,15 +10,13 @@ run
 
 in this directory.
 
-To build honggfuzz, you must have libunwind on your system, as well as
-libopcodes and libbfd from binutils **2.38** on your system. The most
-recently-released binutils 2.39 has changed their API in a breaking way.
+You need honggfuzz installed on your system.
```

Comment:
In 3ea836c9c0615fbcf2beb48d64bc7c52518af452:

This is only true on NixOS. On normal systems `cargo hfuzz` will build honggfuzz for you. I'm actually pretty surprised that you can just install it locally and things just work (if you have the right version of binutils) or just don't (otherwise).

I'm happy to have a simpler `nix-shell` invocation but we should maybe clarify that it's only needed if you get "libbfd" build errors.

We could also maybe mention that you need to run `cargo install honggfuzz` at some point, regardless of nix-shell invocation, but that the provided shellscripts will do this for you.

Reply:
I need to add `cargo install honggfuzz` to the nix shell, actually. I thought the command checks if you have honggfuzz installed and takes that instead of building it from scratch, but I am no longer so convinced. Adding the build dependencies to the shell seems like the safe option.

Reply:
@uncomputable the build process is actually super confusing. When you do `cargo install honggfuzz` it installs the `cargo-honggfuzz` binary which is basically a build script. Then when you do `cargo hfuzz run` it rebuilds rust-simplicity "inside of" honggfuzz, with some extra instrumentation and a nice UI. It's the latter step where the nix-installed honggfuzz seems to be overriding something and allowing the build to succeed.